rootfs/standard/usr/bin/mynode_post_upgrade.sh

```diff
@@ -634,6 +634,40 @@ if should_install_app "joininbox" ; then
     fi
 fi
 
+# Install JoinMarket GUI
+if should_install_app "jm"; then
```

Comment:
Should this be "jmg"?

Reply:
Also, if using the "should_install_app" function, it needs to be listed on the application page so users can choose to install it. This gets set via the variables in the application_info.json file.

Reply:
fixed